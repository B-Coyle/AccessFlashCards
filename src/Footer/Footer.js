import React, {Component} from 'react';
import './Footer.scss';


export default class Footer extends Component {
    constructor(props) {
        super(props);
    }
    render(){
        return (
            <article className='footerContainer'>
                <h3>Want to know more?</h3>
                <h4>Resources on Web Accessibility and ARIA</h4>
                <ul>
                    <li><a href='https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Roles'>Using ARIA: Roles, States, and Properties</a></li>
                    <li><a href='https://www.mcdpartners.com/news/top-5-common-web-accessibility-mistakes/'>Top 5 Most Common Web Accessibility Mistakes</a></li>
                    <li><a href='https://en.wikipedia.org/wiki/Web_accessibility'>Web Accessibility-Wikipedia</a></li>
                    <li><a href='https://www.w3.org/WAI/standards-guidelines/wcag/'>Web Content Accessibility Guidelines (WCAG) Overview</a></li>
                </ul>
           </article>
        )
    }
}