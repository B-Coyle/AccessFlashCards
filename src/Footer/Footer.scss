@import '../Styles/variables.scss';


.footerContainer{
    color: black;
    border: $textBorder;
    background-color: rgba(240, 255, 173, .5);
    width: 55%;
    height: 250px;
}