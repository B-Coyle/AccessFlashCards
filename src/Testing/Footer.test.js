import React from 'react';
import ReactDOM from 'react-dom';
import Footer from '../Footer/Footer';
import { shallow } from 'enzyme'; 

  describe('Footer', () => {
      let wrapper;

      beforeEach(()=> {
          wrapper = shallow(
          );
      });

      it('should match the snapshot with all data passed in', () => {
          expect(wrapper).toMatchSnapshot();
      });


  })