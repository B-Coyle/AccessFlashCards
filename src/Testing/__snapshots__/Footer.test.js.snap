// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot with all data passed in 1`] = `
<footer
  className="footerContainer"
>
  <h3
    className="knowMoreLabel"
  >
    Want to know more?
  </h3>
  <h4
    className="resourcesLabel"
  >
    Resources on Web Accessibility and ARIA
  </h4>
  <ul>
    <li>
      <a
        href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Roles"
      >
        Using ARIA: Roles, States, and Properties
      </a>
    </li>
    <li>
      <a
        href="https://www.mcdpartners.com/news/top-5-common-web-accessibility-mistakes/"
      >
        Top 5 Most Common Web Accessibility Mistakes
      </a>
    </li>
    <li>
      <a
        href="https://en.wikipedia.org/wiki/Web_accessibility"
      >
        Web Accessibility-Wikipedia
      </a>
    </li>
    <li>
      <a
        href="https://www.w3.org/WAI/standards-guidelines/wcag/"
      >
        Web Content Accessibility Guidelines (WCAG) Overview
      </a>
    </li>
  </ul>
</footer>
`;
